cpp/include/cudf/search.hpp

```diff
@@ -129,6 +129,32 @@ std::unique_ptr<column> upper_bound(table_view const& t,
 bool contains(column_view const& col, scalar const& value,
               rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**---------------------------------------------------------------------------*
+ * @brief Identify if elements in the column are contained in elements of a second column.
+ *
+ * For each element of col, identify if that element is contained in in_col
```

Comment:
What is the behavior of null elements in `col`? Is the output nullable?

Reply:
Output copies the null map of the input.  So a null input results in a corresponding null output.

Reply:
This needs to be documented.

Reply:
Added to the doxygen comments